src/extensions/replay/sessionrecording.ts

```diff
@@ -544,6 +586,10 @@ export class SessionRecording {
             })
         }
 
+        if (response.sessionRecording?.urlTriggers) {
```

Comment:
Is this something that should be configurable via the client as well?

Reply:
i think ultimately yes although I like to keep things remote only if there's a chance it will change

and my guess is this helps non-engineers more than engineers since it's a "no-code" trigger so client side maybe not super helpful for them anyway